### DISCUSSION TITLE
Different Organic Map Versions

### DISCUSSION THREAD

**Discussion by nep2ner** [organicmaps/organicmaps#7719] [Q&A] [answered]
I'm confused about the recent app ID change and about what is different between app versions.

What is the difference between the Play Store, F-Droid, and Github release versions?

**Comment by biodranik** [comment, marked as answer]
1. Play Store version has built-in world maps and may use Google Location services for faster location detection over WiFi or cellular networks (it can be disabled in OM settings). Note that Location Services setting disappears if your device doesn't have them installed for whatever reason (degoogled device, for example). Releases may be delayed by Google Reviewers or by partial availability to see if there are any regressions.
2. Github (Web) version is the same as Play Store except it doesn't have built-in World maps, which should be downloaded separately. You can auto-update it using Obtanium. It has a different ID so it can be installed separately with Google, Huawei, or Fdroid versions. And you can easily migrate your bookmarks. Available instantly. Note that Location Services setting disappears if your device doesn't have them installed.
3. FDroid version doesn't have world maps and doesn't have Google Location services library in it. That is why it may take a looong time to detect your position using GPS satellites only. It is usually updated in 3-7 days after Web version, but it may take even more time in case of issues.

**Reply by nep2ner** [reply]
I have 2024-02-06-11-Web and would like to update to the newest web version. From what I could gather, the app ID for Web just recently changed so I can't update, I have to reinstall. I have a lot of different bookmark lists, what is the best way for me to easily migrate them all over to the new web release?

**Reply by nep2ner** [reply]
Thank you for clearing the different versions up for me by the way. On the Web version, I don't see the Google Location Library toggle (but I also don't have Google Play Services installed, so maybe the toggle isn't expected to show).

**Reply by biodranik** [reply]
You can install a new web version in parallel, and then migrate bookmarks there from an older version.

**Reply by biodranik** [reply]
> so maybe the toggle isn't expected to show

Exactly. Updated the description.

**Reply by nep2ner** [reply]
Cool, I see the parallel OM showed up in the Android share view.

**Reply by pastk** [reply]
@biodranik strictly speaking the "Location Services setting disappears" situation could happen with Google Play build also (it could be installed on an ungoogled phone e.g. via aurora or direct apk download).
